fpdf/fpdf.py

```diff
@@ -2570,6 +2571,42 @@ def skew(self, ax=0, ay=0, x=None, y=None):
             )
             yield
 
+    @check_page
+    @contextmanager
+    def mirror(self, angle, origin):
```

Comment:
```suggestion
    def mirror(self, origin, angle):
```
If we want later on to make the `angle` optional, and allow to specify a second point instead,
as @gmischler suggested, we should anticipate and make the origin the first argument.

Also, IMHO, I find it more "intuitive" to have the point first and the angle second.

This will require you to edit the unit tests & documentation you wrote,
so I hope this change is OK for you?

Reply:
Makes sense to me!